final-progII/src/presentation/Screen.java

```diff
@@ -1317,15 +1314,26 @@ public void actionPerformed(ActionEvent e) {
 
 		submitLogin__button.addActionListener(new ActionListener() {
 			public void actionPerformed(ActionEvent e) {
-				String ci = nameLogin__textField.getText();
-				String psswd = passwordLogin__textField.getText();
+				int ci = Integer.parseInt(ciLogin__textField.getText());
+				String password = passwordLogin__textField.getText();
 
-				if (isUserLoggedIn) {
+				try {
+
+					controller.login(ci, password);
+
+				} catch (Exception ex) {
+
+					JOptionPane.showMessageDialog(null, ex.getMessage());
```

Comment:
Add a `if`  for differents Exceptions to the User, for example : If you ingress a letter in the texField ci catch the Exception and give a JOptionPane for that.